src/test/java/uk/gov/hmcts/reform/sscs/helper/HearingsPartiesMappingTest.java

```diff
@@ -23,236 +28,69 @@ void buildHearingPartiesDetails() {
         // TODO Finish Test when method done
     }
 
+    @DisplayName("When language passed in should return correct LOV format")
+    @ParameterizedTest
+    @CsvSource({"Acholi,ach-ach", "Afrikaans,afr-afr", "Akan,aka-aka", "Albanian,alb-alb", "Zaza,zza-zza", "Zulu,zul-zul"})
```

Comment:
if you want to test them all, you can add an actual csv with all the values

Reply:
Sadly can't test them all atm maybe when we do the other ticket when all the languages are done